### ISSUE TITLE
[DOCS] LazyMotion page has incorrect variable name

### ISSUE THREAD

**Issue by mattmischuk** [framer/motion#2780]
### Explanation
Incorrect variable name in docs

### Steps to reproduce

1. Go to https://www.framer.com/motion/lazy-motion/
2. Navigate to `Props`
3. See example

### Expected behaviour

```patch
 // features.js
-import { domAnimation } from "framer-motion"
-export default domAnimation
+import { domAnimations } from "framer-motion"
+export default domAnimations

 // index.js
 import { LazyMotion, m } from "framer-motion"
```

**6. Video or screenshots**
<img width="843" alt="image" src="https://github.com/user-attachments/assets/89ac227c-4e60-4b61-8ee0-e321ce59f1fd">

**Comment by sheikhevan** [comment]
looks fixed to me, someone mark this as closed?

**Comment by mattgperry** [comment]
Publishing a fix now, thanks!

**Comment by mattmischuk** [comment]
@mattgperry I think you did the opposite; it's meant to be `domAnimation` not the plural form. See const name in dist.
https://github.com/framer/motion/blob/5619d5644ffe3d22cb523c54d4b36cf5fec26f60/packages/framer-motion/src/render/dom/features-max.ts#L10

**Comment by mattmischuk** [comment]
@mattgperry hey is the docs open source? Can I contribute the fix?

**Comment by vladstudio** [comment]
Yeah please replace `domAnimations` with `domAnimation` at https://www.framer.com/motion/guide-reduce-bundle-size/
